core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/replication-repmgr-Debian.yml

```diff
@@ -1,185 +1,181 @@
-  changed_when: false
-  register: is_master_already_registered
-  failed_when: is_master_already_registered.rc == 2
-  args:
-    executable: /bin/bash
-  when:
-    - groups['postgresql'][0] == inventory_hostname
-
-- name: Restart postgresql service
-  service:
-    name: "{{ pg_service_name[ansible_os_family] }}"
-    state: restarted
-  when:
-    - groups['postgresql'][0] == inventory_hostname
-    - is_master_already_registered.stdout == ""
-
-- name: Restart repmgrd service
-  service:
-    name: "{{ repmgr_service_name[ansible_os_family] }}"
-    state: restarted
-  when:
-    - groups['postgresql'][0] == inventory_hostname
-    - is_master_already_registered.stdout == ""
-
-- name: Register primary with repmgr
-  become_user: postgres
-  shell: "{{ repmgr_bindir[ansible_os_family] }}/repmgr primary register -f {{ repmgr_config_dir[ansible_os_family] }}/repmgr.conf
-    --force --superuser={{ specification.extensions.replication.priviledged_user_name }}"
-  when:
-    - groups['postgresql'][0] == inventory_hostname
-    - is_master_already_registered.stdout == ""
-
-# Slave:
-- name: Check if node is already registered in repmgr
-  become_user: postgres
-  shell: >-
-    set -o pipefail &&
-    {{ repmgr_bindir[ansible_os_family] }}/repmgr cluster show -f {{ repmgr_config_dir[ansible_os_family] }}/repmgr.conf | grep -i {{ inventory_hostname }}
-  changed_when: false
-  register: is_node_already_registered
-  failed_when: is_node_already_registered.rc == 2
-  args:
-    executable: /bin/bash
-  when:
-    - groups['postgresql'][1] == inventory_hostname
-
-- name: Stop postgresql service
-  service:
-    name: "{{ pg_service_name[ansible_os_family] }}"
-    state: stopped
-  when:
-    - groups['postgresql'][1] == inventory_hostname
-    - is_node_already_registered.stdout == ""
-
-- name: Clone content with repmgr
-  become_user: postgres
-  shell: "{{ repmgr_bindir[ansible_os_family] }}/repmgr -h {{ hostvars[groups['postgresql'][0]]['ansible_default_ipv4']['address'] }} -U {{ specification.extensions.replication.priviledged_user_name }} -d {{ specification.extensions.replication.repmgr_database }} -p 5432 -F standby clone"
-  when:
-    - groups['postgresql'][1] == inventory_hostname
-    - is_node_already_registered.stdout == ""
-
-- name: Start postgresql service
-  service:
-    name: "{{ pg_service_name[ansible_os_family] }}"
-    state: started
-  when:
-    - groups['postgresql'][1] == inventory_hostname
-    - is_node_already_registered.stdout == ""
-
-- name: Restart repmgrd service
-  service:
-    name: "{{ repmgr_service_name[ansible_os_family] }}"
-    state: restarted
-  when:
-    - groups['postgresql'][1] == inventory_hostname
-    - is_node_already_registered.stdout == ""
+# For repmgr installed from Ubuntu package additional configuration is required before repmgrd is started as daemon
+- name: Set repmgr.conf file in /etc/default/repmgrd
+  replace:
```

Comment:
Same argument as in `I don't really like that we're using lineinfile, for me it's ugly..` :hugs: